Notepad-Sharp/MainForm.cs

```diff
@@ -798,13 +799,6 @@ private void MainForm_KeyDown(object sender, KeyEventArgs e)
                         CreateTab(file_open.FileName);
                     }
                 }
-                else if (e.KeyCode == Keys.S && e.Modifiers == Keys.Control)
-                {
-                    if (CurrentTB != null)
-                    {
-                        CallSave(CurrentTB);
-                    }
-                }
```

Comment:
This code block was removed because the save hotkey action is no longer handled by the MainForm, it is now handled by the FastColoredTextBox natively.